src/stores/WalletStore.js

```diff
@@ -35,4 +35,17 @@ export default class WalletStore {
   clearEncryptResult = () => {
     this.encryptResult = undefined;
   }
+
+  @action
+  backupWallet = async () => {
+    try {
+      await axios.post(Routes.api.backupWallet);
+    } catch (error) {
+      const errorObject = {
+        route: Routes.api.backupWallet,
+        message: error.message,
+      };
+      this.error = errorObject;
```

Comment:
FYI, looks like showing this error obj in the `ErrorDialog` has a dependency on doing the MobX refactor for the `ErrorDialog`. the errors are being stored here, but aren't being used. So if backup or encrypt fail it is not being caught.

Reply:
@dwalintukan just check the code, this encrypt doesn't use the `ErrorDialog`, i used `EncryptStatusDialog` to display the success or failed by check the `encryptResult `object

Reply:
@henryzbw hm ok. can you use the `ErrorDialog` if it errors and `EncryptStatusDialog` for success? I want to formalize all errors that need a dialog to use the `ErrorDialog`.